### DISCUSSION TITLE
Welcome to Kooha Discussions!

### DISCUSSION THREAD

**Discussion by SeaDve** [SeaDve/Kooha#174] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MarkG-108** [comment]
Hello!  Kooha is a great initiative with a promising future. .Thanks for taking the initiative to develop Kooha.

I am a long time Linux user who recently bought a Pinephone Pro.  I hope to be able to give some feedback and bug reports on Kooha as a small contribution to its development upon this device.

**Reply by jgarte** [reply]
@MarkG-108 How'd it go with testing Kooha on the Pinephone Pro? Did the UI work well with the Pinephone Pro? All best.

**Comment by rouelle** [comment]
I'm just discovering Kooha after reading this article: https://www.debugpoint.com/screen-recorders-linux-wayland/ 
May I suggest to change locale for French from fr_FR to fr only so that fr_CA or fr_BE can also get the French translation.

**Comment by JwanKhalaf** [comment]
Thanks for your hard work @SeaDve <3

I wanted to double check with you as I couldn't find this in the docs. Is Kooha **Wayland only**?

I'm on X11 (bspwm and polybar) and it doesn't seem to work for me, I made sure I have all the runtime requirements installed. When I click the record button, I get:

```
Failed to start recording

Caused by:
    0: Check out <a href="https://github.com/SeaDve/Kooha#-it-doesnt-work">It Doesn't Work page</a> for help.
    1: Failed to create ScreencastSession
    2: Failed to create session
    3: Failed to call `CreateSession` with parameters: Variant { ptr: 0x640225857e50, type: VariantTy { inner: "(a{sv})" }, value: "({'session_handle_token': <'kooha_0'>, 'handle_token': <'kooha_1'>},)" }
    4: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: No such interface “org.freedesktop.portal.ScreenCast” on object at path /org/freedesktop/portal/desktop
```

I'll continue going down the points mentioned in the [troubleshooting checklist](https://github.com/emersion/xdg-desktop-portal-wlr/wiki/%22It-doesn't-work%22-Troubleshooting-Checklist).

I don't seem to have anything set here, so maybe that's the issue, I'll investigate:

```
~ ❯ echo $XDG_CURRENT_DESKTOP

~ ❯ 
```
